appveyor.yml

```
@@ -0,0 +1,39 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+environment:
+  matrix:
+    - compiler: msvc-15-seh
+      generator: "Visual Studio 15 2017 Win64"
```

```suggestion
  matrix:
    - compiler: msvc-16-seh
      generator: "Visual Studio 16 2019 Win64"

    - compiler: msvc-15-seh
      generator: "Visual Studio 15 2017 Win64"
```

It would be nice testing with the latest MVSC as_MSC_VER is used in Folly.